github4s/shared/src/main/scala/github4s/api/PullRequests.scala

```diff
@@ -28,62 +28,59 @@ import scala.language.higherKinds
 
 /** Factory to encapsulate calls related to PullRequests operations  */
 class PullRequests[C, M[_]](
-  implicit urls: GithubApiUrls,
-  C: Capture[M],
-  httpClientImpl: HttpRequestBuilderExtension[C, M]) {
+    implicit urls: GithubApiUrls,
+    C: Capture[M],
+    httpClientImpl: HttpRequestBuilderExtension[C, M]) {
```

Comment:
could you turn off the auto-indent here (or change it) as we'd like to keep the same indentation across the project?

Reply:
this is true for a lot of the files below too

Reply:
Fixed